partnercenter/azext_partnercenter/_client_factory.py

```diff
@@ -22,31 +22,19 @@ def cf_partnercenter(cli_ctx, *_):
     return None
 
 
+def cf_offers(cli_ctx, *_):
+    from partnercenter.azext_partnercenter.clients import OfferClient
+    client = OfferClient(cli_ctx, *_)
+    return client
+
+
 def cf_plans(cli_ctx, *_):
-    from partnercenter.azext_partnercenter.operations.marketplace_offer_plan._client import PlanClient
+    from partnercenter.azext_partnercenter.clients import PlanClient
     client = PlanClient(cli_ctx, *_)
     return client
 
+
 def cf_list(cli_ctx, *_):
     from partnercenter.azext_partnercenter.operations.marketplace_offer._client import ListClient
     client = ListClient(cli_ctx, *_)
     return client
-
-def get_api_client(cli_ctx, *_):
```

Comment:
This was moved to clients/_util to eliminate all awareness of the generated SDK in the operations and root code, hiding it in the clients, models layer.